data/blog/donating-to-opensats.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: "Navigating Donations to OpenSats"
+date: '2023-09-30'
+tags: ['OpenSats', 'bitcoin', 'fiat', 'donations']
+draft: false
+authors: ['arvin']
+images: ['/static/images/blog/08-nostr-design.jpg']
+summary: "This step-by-step guide offers clear instructions on how to make tax-deductible or anonymous bitcoin and fiat donations to OpenSats, a 501(c)(3) nonprofit advancing financial and censorship freedom through free open-source software."
+---
+
+OpenSats champions free & open-source software (FOSS) to advance financial and censorship freedom worldwide. As a 501(c)(3) public charity, 100% of donations go directly to FOSS projects and contributors. This guide offers clear instructions to easily make tax-deductible or anonymous donations supporting [our mission](https://opensats.org/mission)
+
+You will learn how to:
+
+ 1. Make bitcoin donations that are tax-deductible or anonymous.
+ 2. Make fiat donations using credit/debit cards, wire transfers, or Donor-Advised Funds.
+
+### Donate Bitcoin
+
+OpenSats accepts bitcoin without limitations. Choose to make tax-deductible or anonymous non-tax-deductible donations. Regardless, your bitcoin contributions have tremendous impact, advancing OpenSats' vital work championing financial and censorship freedom.
+
+#### Bitcoin Donations Valued Under $5,000
+
+ 1. Visit [OpenSats' donation page](https://opensats.org/projects) to select a Fund/Project to support.
+ 2. Click on **"Donate"** to complete the short donor information form.
+ 3. Click on **"Donate with Bitcoin"** at the bottom of the form to be redirected to a secure bitcoin payment portal.
+ 4. Send funds using your Bitcoin wallet:
+  - Scan the QR code (Bitcoin or Lightning); OR
+  - Copy/paste the Invoice ID (Lightning) or the Address/Payment Link (Bitcoin).
+  - Note: Donations above $50 have Bitcoin/Lightning options. Donations below $50 will only have Lightning.
```

Comment:
When I look at the preview link (https://os-website-git-donate-daf-open-sats.vercel.app/blog/donating-to-opensats), it seems like wherever there are $ signs placed within the content of the article, the font types gets messed up. Not sure if this is only for the sandbox preview or if the bug will render on the production site. Thank you.

Reply:
The rendering engine uses the [remark-math](https://github.com/remarkjs/remark-math/tree/main/packages/remark-math) plugin, so anything that is between two `$` signs will be [rendered as a math formula](https://mdxjs.com/guides/math/). That's why it looks weird.

You'll have to "[escape](https://en.wikipedia.org/wiki/Escape_character)" the $ character by writing `\$50` instead of `$50`. In other words: just add a backslash (`\`) in front of the `$`.